examples/training_without_local_storage.ipynb

```diff
@@ -0,0 +1,917 @@
+    "    batch_size=batch_size,\n",
+    ")\n",
+    "eval_dataset = MNIST('~/datasets', train=False, download=True, transform=ToTensor())\n",
+    "eval_dataloader = torch.utils.data.DataLoader(\n",
+    "    dataset=eval_dataset,\n",
+    "    batch_size=batch_size,\n",
+    ")\n",
+    "\n",
+    "tensorboard_log_dir = '/tmp/tb_logs'\n",
+    "checkpoint_dir = '/tmp/checkpoints'\n",
+    "shutil.rmtree(tensorboard_log_dir, ignore_errors=True)\n",
+    "shutil.rmtree(checkpoint_dir, ignore_errors=True)\n",
+    "\n",
+    "trainer = Trainer(\n",
+    "    model=model,\n",
+    "    max_duration='2ep',\n",
+    "    # Make training fast: Terminate each epoch after 5 batches\n",
+    "    train_subset_num_batches=5,\n",
+    "    optimizers=optimizer,\n",
+    "    train_dataloader=train_dataloader,\n",
+    "    eval_dataloader=eval_dataloader,\n",
+    "    # Make evaluation fast: Evaluate only on five batches\n",
+    "    eval_subset_num_batches=5,\n",
+    "    # Flush log files every batch\n",
+    "    loggers=TensorboardLogger(log_dir=tensorboard_log_dir, flush_interval=1),\n",
+    "    save_folder=checkpoint_dir,\n",
+    "    save_interval=\"1ba\",\n",
+    "    run_name='local_training_run',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Train!\n",
+    "trainer.fit()\n",
+    "\n",
+    "# Close the trainer\n",
+    "trainer.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We trained our model. Now let's verify that we have our checkpoints, and that we can visualize our Tensorboard logs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Checkpoint Files\")\n",
+    "!ls -al {checkpoint_dir}\n",
+    "\n",
+    "# Visualize Tensorboard Logs\n",
+    "%load_ext tensorboard\n",
+    "import tensorboard.notebook\n",
+    "\n",
+    "%tensorboard --logdir {tensorboard_log_dir}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Storing and loading checkpoints and logs with the cloud"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we verified our local training workflow works as intended, let's configure Composer to back up our checkpoints and Tensorboard TF Event files to the cloud. \n",
+    "\n",
+    "First, a brief overview of Composer architecture:\n",
+    "\n",
+    "* [Logger](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger): Composer includes a centralized logger, which passes logged data, which can either metrics or artifacts, to each `LoggerDestination` (more on that below). It's similar to Python's built-in `logging.getLogger(...)` but is designed to log structured metrics and artifacts rather than unstructured log messages.\n",
```

Comment:
```suggestion
    "* [Logger](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger): Composer includes a centralized logger, which passes logged data, which can be either metrics or artifacts, to each `LoggerDestination` (more on that below). It's similar to Python's built-in `logging.getLogger(...)` but is designed to log structured metrics and artifacts rather than unstructured log messages.\n",
```